Source/JavaScriptCore/inspector/protocol/Page.json

```diff
@@ -119,10 +119,12 @@
                 { "name": "value", "type": "string", "description": "Cookie value." },
                 { "name": "domain", "type": "string", "description": "Cookie domain." },
                 { "name": "path", "type": "string", "description": "Cookie path." },
+                { "name": "partitionKey", "type": "string", "description": "Cookie partition key." },
                 { "name": "expires", "type": "number", "description": "Cookie expires." },
                 { "name": "session", "type": "boolean", "description": "True in case of session cookie." },
                 { "name": "httpOnly", "type": "boolean", "description": "True if cookie is http-only." },
                 { "name": "secure", "type": "boolean", "description": "True if cookie is secure." },
+                { "name": "partitioned", "type": "boolean", "description": "True if cookie is partitioned." },
```

Comment:
NIT: i'd prefer if this was put right before `partitionKey` (or that is put right after this) so that everything related to partitioning is in the same place

it also might be nice to add in the `description` that they're mutually exclusive, since we can assume `partitioned` is `true` if `partitionKey` is provided, and if just `partitioned` is provided without `partitionKey` then that's Web Inspector asking for the `partitionKey` to be generated